### PR TITLE
Fix: project card doesn't show if proj name is "loader" 

### PR DESCRIPTION
**Pull request by roperzh**
This should only take a couple of minutes to review.

Asana ticket: [Dash bug: project card doesn't show if proj name is "loader" ](https://app.asana.com/0/479779791675933/498221072732921)

### CODE REVIEW

**Comment by stristr** [comment]
LGTM and I don't see any things that actually use an id of "loader", but should get a 👍 from somebody who has been with the code since the beginning to be sure.

File: packages/haiku-creator/public/stylesheets/global.css [hunk 1]
```diff
@@ -129,14 +129,6 @@ iframe {
   -webkit-transition: opacity .25s ease-in-out;
 }
 
-#loader {
```
Comment (roperzh) [inline packages/haiku-creator/public/stylesheets/global.css:132]: @matthewtoast any bad feelings about removing this? I traced it back to https://github.com/HaikuTeam/glass/commit/d643796baefbcd05804a5b4b2b818950d7513a45#diff-1804b38ac46271db28843ef316579fe6 but couldn't find where this was applied.

After a quick test, every loader in the app seems to be working fine

**Comment by tsortwehttam** [comment]
@roperzh That is a good change, but we should go a step further and namespace this:

https://github.com/HaikuTeam/mono/blob/master/packages/haiku-creator/src/react/components/ProjectThumbnail.js#L24

So the ids end up like `haiku-project-thumbnail-FooBar` and `haiku-project-thumbnail-Meow` - which will prevent future accidental collisions.

**Comment by roperzh** [comment]
@matthewtoast I did it in https://github.com/HaikuTeam/mono/pull/125/commits/0d0d785e94a0d8f57757675ad3da9e47115c5552 do you want to change the prefix? If yes I'm in!

**Comment by tsortwehttam** [comment]
Nope that's great. Missed that. LGTM.

**Pull request merged**